examples/simple-bot/simple-bot-slack.js

```diff
@@ -37,8 +37,11 @@ slackController.hears(['.*'], ['direct_message', 'direct_mention', 'mention'], f
     if (message.watsonError) {
       console.log(message.watsonError);
       bot.reply(message, "I'm sorry, but for technical reasons I can't respond to your message");
-    } else {
+    } else if (message.watsonData && message.watsonData.output) {
```

Comment:
Is this for the case when `output` is empty?

Reply:
The idea was, for when it's empty or non-existing (e.g. erroneously removed in .after() )

Reply:
Yes

Reply:
Ok! I think now I've adapted it correctly to only check for the presence of the field, and to let empty strings pass. Made the change in all relevant files :)